docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
I will take on building out the MVP for at least run and resize.

Brendan
On Jan 9, 2015 11:50 AM, "bgrant0607" <notifications@github.com> wrote:

> In docs/cli-usage.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3233#discussion-diff-22738509>
> :
>
> > @@ -0,0 +1,96 @@
> > +## Using Kubernetes from the command line
>
> Other useful utilities from community hangout:
>
>    - Pretty printing of endpoints
>    - Service address/port lookup command(s)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3233/files#r22738509>
> .
>